### PR TITLE
configure nginx ingress tcp-services-configmap flag

### PR DESCRIPTION
**Pull request by kopiczko**
Towards https://github.com/giantswarm/giantswarm/issues/1526

### CODE REVIEW

File: templates.go [hunk 1]
```diff
@@ -671,6 +671,7 @@ write_files:
             - /nginx-ingress-controller
             - --default-backend-service=$(POD_NAMESPACE)/default-http-backend
             - --configmap=$(POD_NAMESPACE)/ingress-nginx
+            - --tcp-services-configmap=$(POD_NAMESPACE)/ingress-nginx-tcp-services
```
Comment (kopiczko) [inline templates.go]: I will align this name with https://github.com/giantswarm/g8s-ingress-controller/pull/10.
I wanted it to be aligned with `--configmap=$(POD_NAMESPACE)/ingress-nginx` above.

**Comment by xh3b4sd** [comment]
Why are we doing this here? AFAIK the ingress controller within the guest clusters do not need this, but the ingress controller on the host cluster.

**Comment by JosephSalisbury** [comment]
We're aiming to use k8scloudconfig for host clusters too

**Comment by kopiczko** [comment]
@xh3b4sd very good point. @calvix @r7vme can you point me to the place where host cluster's ingress is configured?

**Comment by r7vme** [comment]
@kopiczko Here https://github.com/giantswarm/hive/blob/master/template_snippets/cloudconfig/g8s_master.yaml#L1022-L1139

**Comment by xh3b4sd** [comment]
> We're aiming to use k8scloudconfig for host clusters too

Fine, but how do we handle differences then? The TCP configmap flag isn't a common thing across guest and host clusters.

**Comment by kopiczko** [comment]
Closing this. This should be changed here: https://github.com/giantswarm/k8scloudconfig/pull/155#issuecomment-308701854

If [this](https://github.com/giantswarm/k8scloudconfig/pull/155#issuecomment-308669006) is something we want, let's track it, and add properly.

**Pull request closed without merging**